src/mdlComponentHandler.js

```diff
@@ -86,6 +86,15 @@ var componentHandler = (function() {
    * the element to.
    */
   function upgradeElementInternal(element, jsClass) {
+    if (jsClass === undefined) {
+      for (var i = 0; i < registeredComponents_.length; i++) {
```

Comment:
Yeah, I was just going for the most-likely-to-be-pulled quickly approach and copy-pasted from how it's done above. =)